src/prefabs/pageWithCreateDataTable.js

```diff
@@ -0,0 +1,12336 @@
+                            type: 'HIDE',
+                            option: 'collapsable',
+                            comparator: 'EQ',
+                            value: true,
+                          },
+                        },
+                      },
+                      {
+                        type: 'CUSTOM',
+                        label: 'Vertical Alignment',
+                        key: 'verticalAlignment',
+                        value: 'stretch',
+                        configuration: {
+                          as: 'BUTTONGROUP',
+                          dataType: 'string',
+                          allowedInput: [
+                            { name: 'Top', value: 'flex-start' },
+                            { name: 'Center', value: 'center' },
+                            { name: 'Bottom', value: 'flex-end' },
+                            { name: 'Justified', value: 'stretch' },
+                          ],
+                        },
+                      },
+                      {
+                        value: ['0rem', '0rem', 'M', '0rem'],
+                        label: 'Outer space',
+                        key: 'outerSpacing',
+                        type: 'SIZES',
+                      },
+                    ],
+                    descendants: [],
+                  },
+                ],
+              },
+            ],
+          },
+        ],
+      },
+    ];
+
+    return (
+      <>
+        <Header title="Configure data table" onClose={close} />
+        <Content>
+          <Field
+            label="Model"
+            error={
+              modelValidation && (
+                <Text color="#e82600">Selecting a model is required</Text>
+              )
+            }
+          >
+            <ModelSelector
+              onChange={value => {
+                setModelValidation(false);
+                setModelId(value);
+              }}
+              value={modelId}
+            />
+          </Field>
+          <Field
+            label="Columns in the data table"
+            error={
+              propertiesValidation && (
+                <Text color="#e82600">Selecting a property is required</Text>
+              )
+            }
+          >
+            <PropertiesSelector
+              modelId={modelId}
+              value={properties}
+              disabledKinds={[
+                'BELONGS_TO',
+                'HAS_AND_BELONGS_TO_MANY',
+                'HAS_MANY',
+                'MULTI_FILE',
+                'AUTO_INCREMENT',
+                'COUNT',
+                'MULTI_IMAGE',
+                'PDF',
+                'RICH_TEXT',
+                'SIGNED_PDF',
+                'SUM',
+                'BOOLEAN_EXPRESSION',
+                'DATE_EXPRESSION',
+                'DATE_TIME_EXPRESSION',
+                'DECIMAL_EXPRESSION',
+                'INTEGER_EXPRESSION',
+                'MINUTES_EXPRESSION',
+                'PRICE_EXPRESSION',
+   
```

Comment:
Your new `autoOK` option needs to be added here as well

Reply:
Added autoOK option